### ISSUE TITLE
Features extraction

### ISSUE THREAD

**Issue by tommyz94** [wizyoung/YOLOv3_TensorFlow#121]
Hi wizyoung,
There's a way to extract features from the last convolutional layer for each bounding box?

Thanks

**Comment by Cli98** [comment]
@tommyz94 what's the meaning of extract feature? Get weights of layer?

**Comment by tommyz94** [comment]
I mean the features extracted by the last convolutional layer. For example in VGG16 after all convolutional phase I can obtain a vector of dimension 4096 that describes the image  at an high-level.

**Comment by Cli98** [comment]
@tommyz94 Yes it is possible, you can hook out weights of layer and save it, from you kept

**Comment by Cli98** [comment]
@tommyz94 Yes it is possible, you can hook out weights of layer and save it, from you checkpoint

**Comment by bujianyiwang** [comment]
I want to use yolov3 to predict the number of real-time person from the rtsp stream with suited interval,anyone has the right python file?

**Comment by bujianyiwang** [comment]
I want to use yolov3 to predict the number of real-time person from the rtsp stream with suited interval,anyone has the right python file?

**Comment by SouradipBh** [comment]
> I mean the features extracted by the last convolutional layer. For example in VGG16 after all convolutional phase I can obtain a vector of dimension 4096 that describes the image at an high-level.

Do u have it?
